model_api.py

```diff
@@ -291,7 +291,15 @@ def process_fmap_grad(grad):
                 # Mask the image
                 masked = (mask[..., np.newaxis] * np_image).astype(np.uint8)
                 word_masks[(chunk.position, chunk.attribute)] = masked
+        elif word_highlights:
+            word_masks = {}
+            visual = np.zeros((224, 224))
+            self.model.zero_grad()
+            log_probs.sum().backward(retain_graph=True)
+            mask = visual
```

Comment:
If you want you can also try with other variations here that change the visual appearance of the mask, such as

```python
mask = visual ** 2
# or
mask = visual ** 0.5
```